index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <div>
+            <nav id="head">
+                <p>Safar Ali</p>
+                <img src="../Portfolio/images/Union.png" alt="burger-icon">
+            </nav>
+            <div id="name">
+                <h2>I'm Safar Ali <br> A thriving <br> Software Developer </h2>
+            </div>
+            <div id="para">
+                <p>I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div id="text">
+                <p>Let us Connect</p>
+            </div>
+            <div id="icons">
+                <ul>
+                    <li><img src="../Portfolio/images/github.png" alt=""></li>
+                    <li><img src="../Portfolio/images/Linkedin.png" alt=""></li>
+                    <li><img src="../Portfolio/images/twitter.png" alt=""></li>
+                   <li><img src="../Portfolio/images/Vector.png" alt=""></li>
```

Comment:
- [ ] Great job so far, you're almost there. Each icon in the list should have a link. Please, add an anchor <a> tag to add those links. let's rock, fix them 😃